myst_nb/__init__.py

```diff
@@ -66,8 +66,9 @@ def save_glue_cache(app, env):
 
 def setup(app):
     """Initialize Sphinx extension."""
-    # Sllow parsing ipynb files
-    app.add_source_suffix(".ipynb", "ipynb")
+    # Allow parsing ipynb files
+    app.add_source_suffix(".md", "myst-nb")
```

Comment:
These are overridden when people manually specify this in `conf.py`, yeah?

Reply:
Yes, as in if you do:

```python
source_suffix = {
   ".ext": "parser_type"
}
```

then now only `.ext` would be read by sphinx. I changed `ipynb` to `myst-nb` because now obviously this covers more than just `.ipynb` extensions

Reply:
makes sense to me 👍